app/src/ui/changes/continue-rebase.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import { Button } from '../lib/button'
+import { Loading } from '../lib/loading'
+import { RebaseConflictState } from '../../lib/app-state'
+import { Dispatcher } from '../dispatcher'
+import { Repository } from '../../models/repository'
+import { WorkingDirectoryStatus } from '../../models/status'
+import { getConflictedFiles } from '../../lib/status'
+
+interface IContinueRebaseProps {
+  readonly dispatcher: Dispatcher
+  readonly repository: Repository
+  readonly workingDirectory: WorkingDirectoryStatus
+  readonly rebaseConflictState: RebaseConflictState
+  readonly isCommitting: boolean
+}
+
+export class ContinueRebase extends React.Component<IContinueRebaseProps, {}> {
+  private onSubmit = () => {
+    this.continueRebase()
+  }
+
+  private async continueRebase() {
+    await this.props.dispatcher.continueRebase(
+      this.props.repository,
+      this.props.workingDirectory
+    )
+  }
+
+  public render() {
+    const { manualResolutions } = this.props.rebaseConflictState
+
+    let canCommit = true
+
+    let tooltip = 'Continue rebase'
+
+    if (this.props.rebaseConflictState) {
```

Comment:
**Non-blocking question**
What are we checking for here or is this check due to a limitation in the TS compiler? I'm asking because the type signature for this property is `RebaseConflictState`.

Reply:
This is an unnecessary check - dropped in d3a55eccb